### ISSUE TITLE
space key bypass user defined keymaps, simply be ignored

### ISSUE THREAD

**Issue by t9md** [atom/atom#12953]
### Prerequisites

* [x] Can you reproduce the problem in [safe mode](http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode)?
* [x] Are you running the [latest version of Atom](http://flight-manual.atom.io/hacking-atom/sections/debugging/#update-to-the-latest-version)?
* [x] Did you check the [debugging guide](http://flight-manual.atom.io/hacking-atom/sections/debugging/)?
* [x] Did you check the [FAQs on Discuss](https://discuss.atom.io/c/faq)?
* [x] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
* [x] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if your bug or enhancement is already reported?

### Description

I'm using macOS 10.11.6
In beta

```
Atom    : 1.12.0-beta0
Electron: 1.3.6
Chrome  : 52.0.2743.82
Node    : 6.3.0
```

`space` key is just ignored my keymap.cson, 

keymap.cson to reproduce

```
'atom-text-editor':
  'space': 'core:move-right'
  'i': 'core:move-right'
```

`i` moves cursor to right, but `space` just insert single '` `'(space).
keybinding-resolver say nothing against my `space` stroke.

Stable atom have no issue.

**Comment by thomasjo** [comment]
@nathansobo Do we need to expand [`NON_CHARACTER_KEY_NAMES_BY_KEYBOARD_EVENT_KEY`](https://github.com/atom/atom-keymap/blob/83a451b4ea71b441d42d24611910292f61d1ea36/src/helpers.coffee#L7-L14) with support for the space key?

**Comment by jonasws** [comment]
Experiencing this as well. I'm probably not the only one trying to get the "best of both worlds" experience by using space as a "leader key", to toggle the Command Palette, ala [Spacemacs](http://spacemacs.org/)

**Comment by nathansobo** [comment]
This one was an oversight. We should have a hotfix on beta later today. Thanks to everyone for taking the time to report it.

**Comment by t9md** [comment]
Thanks for very quick fix!!

**Comment by nathansobo** [comment]
This should be fixed in 1.12.0-beta1, which we just released. Please let me know if I missed anything.
